app/src/main/AndroidManifest.xml

```diff
@@ -94,7 +96,8 @@
             android:syncable="false"/>
 
         <activity android:name=".activities.CompassCalibrationActivity"
-                  android:launchMode="singleTop">
+            android:launchMode="singleTop"
+            android:label="Calibration">
```

Comment:
Sorry - I think these would need to be localized. I can't just use English strings.